### ISSUE TITLE
what's the name of this software?

### ISSUE THREAD

**Issue by das-g** [bpeel/prevodb#3]
If I package this software for a Linux distribution, **what should I call the package**? It'd be one _single_ package that'd provide both command line programs, `prevo` **and** `prevodb`.

While this repo is called "prevodb", I somewhat feel like `prevo` is the core piece of the software (from a user's point of view, not regarding amount or difficulty of code) and `prevodb` "just" an additional tool needed to prepare the data for `prevo`, so the package, too, should be called just `prevo`. (I guess you named the repo that way to distinguish it from the one of our Android app of the same name.) But I'd like to hear your opinion as the software's author before deciding for one package name or the other.

**Comment by bpeel** [comment]
I agree with your analysis and also think it would be better to call it just `prevo`. I guess hardly anybody will be interested in using the `prevodb` tool and the `prevo` tool is much more interesting. I would think it could make sense to make a separate package for each tool, but I don’t have a strong opinion so feel free to do whatever you think is best. Will you include the data from ReVo too in the package? Or make a separate package for it? I could potentially make tagged releases of that too. I keep a copy of it in a repo [here](https://github.com/bpeel/revo).

**Comment by das-g** [comment]
> I would think it could make sense to make a separate package for each tool, but I don’t have a strong opinion so feel free to do whatever you think is best.

I thought I'd make it a single package, because it's a similar situation to that of `locate` and `updatedb` which are also usually provided together in a package. (Also, that way I don't have to figure out how to build the programs separately. :wink:)

> Will you include the data from ReVo too in the package? Or make a separate package for it?

I'll make a separate package for the data (which will require the other package for building) and maybe another package that simply pulls in both the software and the data package.

I haven't settled on names for the data package and the third package yet. Maybe `prevo-data` and `prevo-full`?

**Comment by das-g** [comment]
B.t.w., here's my work in progress for the packaging: https://github.com/NixOS/nixpkgs/compare/master...das-g:prevo-init?expand=1
